src/routes/docs/quick-starts/go/+page.markdoc

```diff
@@ -0,0 +1,340 @@
+![Create project screen](/images/docs/quick-starts/integrate-server.png)
+{% /only_light %}
+
+| Category  {% width=120 %} | Required scopes       | Purpose |
+|-----------|-----------------------|---------|
+| Database  | `databases.write`     | Allows API key to create, update, and delete [databases](/docs/products/databases/databases). |
+|           | `collections.write`   | Allows API key to create, update, and delete [collections](/docs/products/databases/collections). |
+|           | `attributes.write`    | Allows API key to create, update, and delete [attributes](/docs/products/databases/collections#attributes). |
+|           | `documents.read`      | Allows API key to read [documents](/docs/products/databases/documents). |
+|           | `documents.write`     | Allows API key to create, update, and delete [documents](/docs/products/databases/documents). |
+
+Other scopes are optional.
+
+{% /section %}
+{% section #step-2 step=2 title="Create Go project" %}
+
+Create a go application.
+
+```sh
+mkdir my-app
+cd my-app
+go mod init go-appwrite/main
+```
+
+{% /section %}
+{% section #step-3 step=3 title="Install Appwrite" %}
+
+Install the Go Appwrite SDK.
+
+```sh
+go get github.com/appwrite/sdk-for-go@1.6.x
+```
+
+Your `go.mod` file should look like this:
+
+```go
+module go-appwrite/main
+
+require github.com/appwrite/sdk-for-go v0.0.0-20240703150753-748203e5536c
+
+go 1.22.5
+```
+
+If it doesn’t, go to your `go.mod` file and copy and paste the above code into the file.
+
+Make sure your `go.sum` file looks like this:
+
+```go
+github.com/appwrite/sdk-for-go v0.0.0-20240703150753-748203e5536c h1:dT+W+Ujz5cuaEvOe8uVfy0xox8YduBJR6uECkgMOEc8=
+github.com/appwrite/sdk-for-go v0.0.0-20240703150753-748203e5536c/go.mod h1:lLim0S47hdXlLet+GAayvBC7VfVxcsoSZ1F5oymXrmc=
+github.com/open-runtimes/types-for-go/v4 v4.0.0 h1:eMKlziaQVPVtgs2VSi1V/grdw3iGbOca/YrHIwMNUXk=
+github.com/open-runtimes/types-for-go/v4 v4.0.0/go.mod h1:88UUMYovXGRbv5keL4uTKDYMWeNtIKV0BbxDRQ18/xY=
+```
+
+If it doesn’t, go to your `go.sum` file and copy and paste the above code into the file.
+
+{% /section %}
+
+{% section #step-4 step=4 title="Import Appwrite" %}
+
+Find your project ID in the **Settings** page. Also, click on the **View API Keys** button to find the API key that was created earlier.
+
+{% only_dark %}
+![Project settings screen](/images/docs/quick-starts/dark/project-id.png)
+{% /only_dark %}
+{% only_light %}
+![Project settings screen](/images/docs/quick-starts/project-id.png)
+{% /only_light %}
+
+Create a new file called `app.go`, initialize a function, and initialize the Appwrite Client. Replace `<YOUR_PROJECT_ID>` with your project ID and `<YOUR_API_KEY>` with your API key.
+
+```go
+package main
+
+import (
+	"fmt"
+
+	"github.com/appwrite/sdk-for-go/client"
+	"github.com/appwrite/sdk-for-go/databases"
+)
+
+func initClient() {
+	appwriteClient := client.NewClient()
+	appwriteClient.SetProject("<PROJECT_ID>")
+	appwriteClient.SetKey("<API_KEY>")
```

Comment:
Looks like the Go SDK had a big update. See https://github.com/appwrite/sdk-for-go/blob/1.6.x/README.md